packages/bpk-component-chip-group/src/BpkChipGroup.tsx

```diff
@@ -0,0 +1,254 @@
+  hidden?: boolean;
+} & SingleSelectChipItem;
+
+export type InternalProps = {
+  chipGroupClassNames: string
+};
+
+export type CommonProps = {
+  label?: string;
+  ariaLabel?: string;
+  chipStyle?: ChipStyleType;
+  chips: ChipItem[];
+  ariaMultiselectable?: boolean;
+};
+
+export type RailChipGroupProps = {
+  stickyChip?: ChipItem;
+  leadingNudgerLabel: string;
+  trailingNudgerLabel: string;
+} & CommonProps;
+
+export type WrapChipGroupProps = {
+} & CommonProps;
+
+export type ChipGroupProps = (RailChipGroupProps & { type: typeof CHIP_GROUP_TYPES.rail } | WrapChipGroupProps & { type: typeof CHIP_GROUP_TYPES.wrap });
+
+const Chip = ({ ariaMultiselectable, chipIndex, chipItem, chipStyle }: { chipIndex: number, chipItem: ChipItem, chipStyle: ChipStyleType, ariaMultiselectable: boolean }) => {
+  const {
+    accessibilityLabel,
+    component = CHIP_COMPONENT.selectable,
+    hidden = false,
+    leadingAccessoryView = null,
+    onClick,
+    selected,
+    text,
+    ...rest
+  } = chipItem;
+  const Component = CHIP_COMPONENT_MAP[component];
+  return hidden ? null : (
+    <Component
+      key={text}
+      selected={selected ?? false}
+      type={chipStyle}
+      accessibilityLabel={accessibilityLabel || text}
+      onClick={() => {
+        if (onClick) {
+          onClick(!selected, chipIndex);
+        }
+      }}
+      role={ariaMultiselectable ? 'checkbox' : 'radio'}
+      leadingAccessoryView={leadingAccessoryView}
+      {...rest}
+    >
+      {text}
+    </Component>
+  );
+}
+
+const ChipGroupContent = (
+  { ariaLabel,
+    ariaMultiselectable,
+    chipGroupClassNames,
+    chipStyle,
+    chips,
+    label }:
+    {
+      chipGroupClassNames: string,
+      ariaMultiselectable: boolean,
+      ariaLabel?: string,
+      label?: string,
+      chips: ChipItem[],
+      chipStyle: ChipStyleType
+    }) => (
+  <fieldset
+    className={chipGroupClassNames}
+    role={ariaMultiselectable ? 'group' : 'radiogroup'}
+  >
+    {ariaLabel && <legend className='visually-hidden'>{ariaLabel}</legend>}
+    {label && <BpkText textStyle={TEXT_STYLES.footnote} aria-hidden>{label}</BpkText>}
+    {chips.map((chip, index) => <Chip chipItem={chip} chipStyle={chipStyle} ariaMultiselectable={ariaMultiselectable} chipIndex={index} />)}
+  </fieldset>
+);
+
+const RailChipGroup = ({
+  ariaLabel,
+  ariaMultiselectable = true,
+  chipGroupClassNames,
+  chipStyle = CHIP_TYPES.default,
+  chips,
+  label,
+  leadingNudgerLabel = '',
+  stickyChip,
+  trailingNudgerLabel = '',
+}: RailChipGroupProps & InternalProps) => {
+  const scrollContainerRef = useRef<HTMLElement | null>(null);
+
+  const stickyChipContainerClassnames = getClassName(
+    'bpk-sticky-chip-container',
+    `bpk-sticky-chip-container--${chipStyle}`,
+  );
+
+  return (
+    <>
+      <BpkBreakpoint query={BREAKPOINTS.ABOVE_TABLET}>
+        <Nudger leading ariaLabel={leadingNudgerLabel} chipStyle={chipStyle} scrollContainerRef={scrollContainerRef} />
```

Comment:
The aria label is required in the Nudger but is marked as optional here and we would pass through an empty string which would cause A11y issues? 

This looks to conflict with the APIs, so we should align them - preference on making it required here instead of passing in a blank string

Reply:
Good catch, this originally optional but should now be required so i've removed the default